Update active-directory-privileged-identity-management-how-to-configu…

hanki-microsoft opened the pull request:
…re-security-alerts.md

Added alert severity and description of severity types high/med/low.

azurecla commented:
Hi __@pablissima__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (paulettm). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

hanki-microsoft committed:
Update active-directory-privileged-identity-management-how-to-configure-security-alerts.md

Added alert severity and description of severity types high/med/low.

hanki-microsoft committed:
Update active-directory-privileged-identity-management-how-to-configure-security-alerts.md

Updated  roles activated too frequently and roles don't require MFA to medium not low
